File: Python/SingleNumber_136.py
class Solution:
    def singleNumber(self, nums: list[int]) -> int:
        a=0
        for i in nums:
            a^=i
        return a
            
        
s= Solution()

n= int(input("Enter number of test cases: "))
for i in range(n):
    l=list(map(int,input("Enteer array elements : ").split()))
    print(f"Single Number : {s.singleNumber(l)}")

File: LeetCode/Java/SingleNumber_136.java
import java.util.Scanner;

class Solution_136 {
    public int singleNumber(int[] nums) {
        int result = 0;
        for (int num : nums) {
            result ^= num;
        }
        return result;
    }
}

class SingleNumber_136 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Solution_136 obj = new Solution_136();

        System.out.println("Enter number of test cases: ");
        int n = sc.nextInt();
        sc.nextLine();

        for (int i = 0; i < n; i++) {
            System.out.println("Enter array elements with space separation: ");
            String[] input = sc.nextLine().split(" ");
            int[] array = new int[input.length];
            for (int j = 0; j < input.length; j++) {
                array[j] = Integer.parseInt(input[j]);
            }

            int result = obj.singleNumber(array);
            System.out.printf("Unique Number : %d%n", result);
        }

        sc.close();
    }
}
